frontend/packages/kubevirt-plugin/integration-tests-cypress/tests/action/migration.ts

```diff
@@ -0,0 +1,110 @@
+import { TEMPLATE_NAME, VM_ACTION, VM_STATUS } from '../../const/index';
+import { ProvisionSource } from '../../enums/provisionSource';
+import { testName } from '../../support';
+import { VirtualMachineData } from '../../types/vm';
+import { actionButtons, detailsTab, errorAlert } from '../../view/selector';
+import { tab } from '../../view/tab';
+import { virtualization } from '../../view/virtualization';
+import { action, vm, waitForStatus } from '../../view/vm';
+
+const vmData: VirtualMachineData = {
+  name: `test-vm-migration-${testName}`,
+  namespace: testName,
+  template: TEMPLATE_NAME,
+  provisionSource: ProvisionSource.URL,
+  pvcSize: '1',
+  sshEnable: false,
+  startOnCreation: false,
+};
+
+describe('Test VM Migration', () => {
+  before(() => {
+    cy.Login();
+    cy.visit('/');
+    cy.createProject(testName);
+    vm.create(vmData);
+    virtualization.vms.visit();
+    waitForStatus(VM_STATUS.Off);
+  });
+
+  after(() => {
+    cy.deleteResource({
+      kind: 'VirtualMachine',
+      metadata: {
+        name: vmData.name,
+        namespace: vmData.namespace,
+      },
+    });
+  });
+
+  it('ID(CNV-2140) Migrate VM action button is displayed appropriately', () => {
```

Comment:
This looks like two different tests: 

1. Migrate and cancel buttons aren't displayed when VM is off
2. Migrate buttons is displayed when VM is running

Reply:
split it into 2 tests